ci/deps_parser.py

```diff
@@ -0,0 +1,96 @@
+# Usage: deps_parser.py --deps <DEPS file> --output <flattened deps>
```

Comment:
Needs copyright header.

Reply:
Added the copyright header. Do we also need to update the licenses scripts to check python files?

Reply:
The license script is only for checking code that might ship. The `ci` directory [is excluded](https://github.com/flutter/engine/blob/main/tools/licenses/lib/main.dart#L2157).